homeassistant/components/fibaro/sensor.py

```diff
@@ -27,7 +28,13 @@
         "mdi:fire",
         None,
     ],
-    "CO2": ["CO2", CONCENTRATION_PARTS_PER_MILLION, "mdi:cloud", None],
+    "CO2": [
+        "CO2",
+        CONCENTRATION_PARTS_PER_MILLION,
+        "mdi:cloud",
```

Comment:
Remove the custom icon when setting device class. Ie default the icon to `None`.

Reply:
Good catch! Removed :)